sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statements.scala

```diff
@@ -362,37 +367,48 @@ case class ShowTableStatement(
  * A CREATE NAMESPACE statement, as parsed from SQL.
  */
 case class CreateNamespaceStatement(
-    namespace: Seq[String],
+    namespace: LogicalPlan,
```

Comment:
I'm not very sure about this. For `CREATE` commands, they only need to know the catalog, but no need to lookup table or namespace. I think we can keep using `Seq[String]` and still handle the commands in `ResolveCatalogs`/`ResolveSessionCatalogs`.

Reply:
yea you are right, and in this case, it will be consistent for tables as well.